README.md

```diff
@@ -87,3 +87,20 @@ vim.g.asyncrun_program = vim.empty_dict()
 vim.api.nvim_command("let g:asyncrun_program.vcvars64 = { opts -> '\"C:/Program Files (x86)/Microsoft Visual Studio/2019/Community/VC/Auxiliary/Build/vcvars64.bat\" && ' .. opts.cmd }")
 vim.g.cmake_asyncrun_options = vim.tbl_extend('force', vim.g.cmake_asyncrun_options, {program = 'vcvars64'})
 ```
+
+## DAP configuration example
+
+You can customize the DAP configurations via `g:cmake_dap_configurations`:
```

Comment:
Could you also specify the default values in the table above?

Reply:
The default value of the variable should be `nil` or empty table since your hard-coded configuration works well with adapters with executable type.

Reply:
Done.

Reply:
I think that we could store the default table in `g:cmake_dap_configurations`.